### PR TITLE
Update antismash recipe

### PR DESCRIPTION
**Pull request by cmonjeau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/antismash/meta.yaml [hunk 1]
```diff
@@ -16,10 +15,10 @@ source:
 
 requirements:
   build:
-    - python
+    - python <3.0
```
Comment (bgruening) [inline recipes/antismash/meta.yaml]: Thanks a lot for working on this @cmonjeau!
This pinning should be replaced by the `skip: True # [not py27]` from above, so I think the former was correct.

**Comment by bgruening** [comment]
Is it too big? Oo.
Could you also include such a script https://github.com/bioconda/bioconda-recipes/blob/master/recipes/weeder/pre-unlink.sh to remove the data with `conda remove`

**Comment by abretaud** [comment]
Yes, the reference data is big, I think it's 3Gb downloaded by post-link.sh...
I told @cmonjeau to make this last test before giving up :) It's copy-pasted from https://github.com/galaxy-genome-annotation/docker-galaxy-genome-annotation/blob/master/.travis.yml#L17
I guess it's failing before pre-unlink.sh is called for now, but yes if it works we'll add it

**Comment by bgruening** [comment]
Thanks for all your efforts!

**Comment by abretaud** [comment]
Damn the freed space wasn't enough... I have no other idea in mind

**Comment by johanneskoester** [comment]
We have made changes to our build system (migrated from Travis CI to Circle CI). Please update your fork with the latest upstream/master.

**Comment by bgruening** [comment]
@cmonjeau could you rebase against latest master, maybe we have more luck with CCI.

**Comment by bgruening** [comment]
@cmonjeau thanks a lot! :)

**Comment by cmonjeau** [comment]
@bgruening Thanks for the last step, I was in day off

**Comment by bgruening** [comment]
Thanks to you @cmonjeau!

**Pull request merged**